src/models/FailedLookups.ts

```diff
@@ -0,0 +1,30 @@
+import * as mongoose from 'mongoose';
+import { Schema, Document } from 'mongoose';
+
+mongoose.set('useCreateIndex', true);
+
+interface FailedLookupsInterface extends Document {
```

Comment:
The interface and Schema don't seem to share parity here, how come? Could they be non required fields on the Schema?

Reply:
Fixing